.watchmanconfig

```diff
@@ -1 +0,0 @@
-{}
```

Comment:
careful deleting this file, it's generated by react-native so I think it might be safe to keep around